azure-pipelines.yml

```diff
@@ -6,49 +6,64 @@ trigger:
     exclude:
     - README.md
 
-container:
-  image: ros:dashing
-  options: "--name ci-container -v /usr/bin/docker:/tmp/docker:ro"
+jobs:
+- job: build_cross_image
+  steps:
+    - script: |
+        sudo apt-get update && sudo apt-get install -y --no-install-recommends qemu-user-static
+      displayName: Install QEMU static emulator
+    - script: |
+        docker build --build-arg AUTOWARE_DOCKER_ARCH=arm64v8 --build-arg AUTOWARE_TARGET_ARCH=aarch64 -t performance_test_cross -f tools/Dockerimage.crossbuild .
```

Comment:
Could we change the variables names to not be "AUTOWARE"?

Reply:
Makes total sense, I just reused what I did for Autoware :laughing: